src/BlockEntities/DropSpenserEntity.cpp

```diff
@@ -75,7 +75,10 @@ void cDropSpenserEntity::DropSpense(cChunk & a_Chunk)
 	if (SlotsCnt == 0)
 	{
 		// Nothing in the dropspenser, play the click sound
-		m_World->BroadcastSoundEffect("random.click", m_PosX * 8, m_PosY * 8, m_PosZ * 8, 1.0f, 1.2f);
+		m_World->BroadcastSoundEffect("random.click", static_cast<double>(m_PosX),
+		                                              static_cast<double>(m_PosY),
+		                                              static_cast<double>(m_PosZ),
+													   1.0f, 1.2f);
```

Comment:
May I suggest the following layout:

```cpp
m_World->BroadcastSoundEffect(
	"random.click",
	static_cast<double>(m_PosX),
	static_cast<double>(m_PosY),
	static_cast<double>(m_PosZ),
	1.0f,
	1.0f
);
```

Though really, I personally wouldn't mind
```cpp
m_World->BroadcastSoundEffect("random.click", static_cast<double>(m_PosX), static_cast<double>(m_PosY), static_cast<double>(m_PosZ), 1.0f, 1.0f);
```